packages/taquito-local-forging/src/taquito-local-forging.ts

```diff
@@ -30,19 +36,27 @@ export function getCodec(codec: CODEC, proto: ProtocolsHash) {
         return decodersProto12[codec](consumer) as any;
       },
     };
-  } else {
+  } else if (ProtoInferiorTo(proto, ProtocolsHash.Psithaca2)) {
```

Comment:
will this be something we need to update in the future with new protocol releases?

Reply:
Yes, when there are breaking changes regarding forging between the protocols, we will need to support protocols we are using in the integration tests

Reply:
I see. I am approving, but I think it might be worth looking into streamlining this as a config so that we won't have to update the code each time a new protocol drops